<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GestionServeurs</class>
 <widget class="QDialog" name="GestionServeurs">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>461</width>
    <height>341</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>461</width>
    <height>341</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>461</width>
    <height>341</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Gestionnaire de Mots de passe - Gestion des serveurs</string>
  </property>
  <widget class="QLabel" name="titrePage">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>421</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Gestion des Serveurs</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>441</width>
     <height>291</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="page1">
    <widget class="QPushButton" name="supprimer">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>170</y>
       <width>141</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>Supprimer le serveur</string>
     </property>
    </widget>
    <widget class="QListWidget" name="listeServeurs">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>261</width>
       <height>241</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="titreSelection">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>80</y>
       <width>141</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
       <weight>75</weight>
       <bold>true</bold>
       <underline>false</underline>
      </font>
     </property>
     <property name="text">
      <string>Sélection</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="creer">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>10</y>
       <width>141</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>Créer un serveur</string>
     </property>
    </widget>
    <widget class="QPushButton" name="modifier">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>110</y>
       <width>141</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>Modifier le serveur</string>
     </property>
    </widget>
    <widget class="QPushButton" name="actualiser">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>260</y>
       <width>81</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Actualiser</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page2">
    <widget class="QPushButton" name="annulerServeur">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>90</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Annuler</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="hote">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>90</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="validerServeur">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>60</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Valider</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelNom">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>60</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Nom complet :</string>
     </property>
    </widget>
    <widget class="QLabel" name="titreCreerModifier">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>221</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Créer|Modifier un serveur</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="nom">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>60</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="labelHote">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>90</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Hôte serveur :</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <tabstops>
  <tabstop>creer</tabstop>
  <tabstop>listeServeurs</tabstop>
  <tabstop>actualiser</tabstop>
  <tabstop>modifier</tabstop>
  <tabstop>supprimer</tabstop>
  <tabstop>nom</tabstop>
  <tabstop>hote</tabstop>
  <tabstop>validerServeur</tabstop>
  <tabstop>annulerServeur</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
